diff --git a/dist/esm/index.js b/dist/esm/index.js
index ed00e361f71e185b4a1ac6d557b61d095969f951..3fa11540a413d8a7cf08db7368511ab4f7e0f1eb 100644
--- a/dist/esm/index.js
+++ b/dist/esm/index.js
@@ -5519,7 +5519,7 @@ const getBackgroundColor = (inverse, disabled, darkMode) => {
             ? "background.selectedInverseLighter"
             : "background.containerInverse";
     }
-    return disabled ? "background.container" : "background.body";
+    return disabled ? "background.container" : "background.containerLighter";
 };
 const SelectButton = ({ ...props }) => {
     const { id, open, variant, hoverValue, value, multiSelectValues, disabled, inverse, invalid, border, width, children, theme: theme$1, ariaLabel, darkMode, } = props;
@@ -5814,7 +5814,7 @@ const Dropdown = ({ id, options, open, variant, inverse, width, menuPlacement, w
                     ? "black70"
                     : inverse
                         ? "background.containerInverse"
-                        : "background.body",
+                        : "background.containerLighter",
                 border: `1px solid ${darkMode ? "black50" : selectedTheme.colors.input.default.border}`,
                 borderRadius: "inc30",
                 boxSizing: "border-box",
@@ -6219,7 +6219,7 @@ var ETrackingEvents;
     ETrackingEvents["SEARCH"] = "Search";
 })(ETrackingEvents || (ETrackingEvents = {}));
 const NAV_HEIGHT = "95px";
-const NAV_BACKGROUND_LIGHT = "background.body";
+const NAV_BACKGROUND_LIGHT = "background.containerLighter";
 const NAV_TEXT_PRIMARY = "text.default";
 const NAV_TEXT_SECONDARY = "text.secondary";
 const NAV_FONT_FAMILY = "body";
@@ -7007,7 +7007,7 @@ const LanguageListOptions = ({ options, open, selectedValue, onSelect, customSty
                 fontSize: "inc20",
                 color: darkMode ? "black00" : "text.default",
                 fontFamily: theme$1.fonts.body,
-                bg: darkMode ? "black70" : "black10",
+                bg: darkMode ? "black70" : "black00",
                 boxSizing: "border-box",
                 overflow: "auto",
                 ...customStyles,
diff --git a/dist/index.js b/dist/index.js
index 100dd878266aecfdb99e5d16625de01da01c5697..43159654a74aff23f0158f2b0e28b606da044bb9 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -5521,7 +5521,7 @@
               ? "background.selectedInverseLighter"
               : "background.containerInverse";
       }
-      return disabled ? "background.container" : "background.body";
+      return disabled ? "background.container" : "background.containerLighter";
   };
   const SelectButton = ({ ...props }) => {
       const { id, open, variant, hoverValue, value, multiSelectValues, disabled, inverse, invalid, border, width, children, theme, ariaLabel, darkMode, } = props;
@@ -5816,7 +5816,7 @@
                       ? "black70"
                       : inverse
                           ? "background.containerInverse"
-                          : "background.body",
+                          : "background.containerLighter",
                   border: `1px solid ${darkMode ? "black50" : selectedTheme.colors.input.default.border}`,
                   borderRadius: "inc30",
                   boxSizing: "border-box",
@@ -6221,7 +6221,7 @@
       ETrackingEvents["SEARCH"] = "Search";
   })(ETrackingEvents || (ETrackingEvents = {}));
   const NAV_HEIGHT = "95px";
-  const NAV_BACKGROUND_LIGHT = "background.body";
+  const NAV_BACKGROUND_LIGHT = "background.containerLighter";
   const NAV_TEXT_PRIMARY = "text.default";
   const NAV_TEXT_SECONDARY = "text.secondary";
   const NAV_FONT_FAMILY = "body";
@@ -7009,7 +7009,7 @@
                   fontSize: "inc20",
                   color: darkMode ? "black00" : "text.default",
                   fontFamily: theme__default$1["default"].fonts.body,
-                  bg: darkMode ? "black70" : "black10",
+                  bg: darkMode ? "black70" : "black00",
                   boxSizing: "border-box",
                   overflow: "auto",
                   ...customStyles,
